config/pipeline.yaml

```
@@ -342,9 +361,91 @@ platforms:
 
   shell:
 
+  kontron-kswitch-d10-mmt-6g-2gs:
+    mach: at91
+    class: arm-dtb
+    dtb: 'lan966x-kontron-kswitch-d10-mmt-6g-2gs.dtb'
+    boot_method: uboot
+
+  kontron-kswitch-d10-mmt-8g:
+    mach: at91
+    class: arm-dtb
+    dtb: 'lan966x-kontron-kswitch-d10-mmt-8g.dtb'
+    boot_method: uboot
+
+  imx6dl-riotboard:
+    mach: imx
+    class: arm-dtb
+    boot_method: barebox
+    filters:
+      - blocklist:
+          defconfig:
+            - 'imx_v4_v5_defconfig'
+            - 'multi_v5_defconfig'
+            - 'allmodconfig'
```

```suggestion
    rules:
      defconfig:
        - '!imx_v4_v5_defconfig'
        - '!multi_v5_defconfig'
        - '!allmodconfig'
```

That's the same filters, only translated to the new `rules:` parameters.